### PR TITLE
ensure decorated jobs for kubernetes-security/kubernetes use the private bucket

### PR DESCRIPTION
**Pull request by BenTheElder**
not sure about `default_org:` correct value or if it even matters much in this context, the bucket is definitely the important part

### CODE REVIEW

File: config/jobs/kubernetes-security/generated-security-jobs.yaml [hunk 1]
```diff
@@ -460,13 +489,18 @@ presubmits:
     decorate: true
     decoration_config:
       gcs_configuration:
-        bucket: kubernetes-jenkins
+        bucket: kubernetes-security-prow
```
Comment (BenTheElder) [inline config/jobs/kubernetes-security/generated-security-jobs.yaml]: /priority critical-urgent

**Comment by BenTheElder** [comment]
cc @liggitt @spiffxp

**Comment by BenTheElder** [comment]
there might be other issues with these configs, and probably the setup should be overhauled or turned down, but this is the immediately glaringly obvious problem

**Comment by liggitt** [comment]
looks reasonable to my uninformed eyes

File: config/jobs/kubernetes-security/genjobs.go [hunk 2]
```diff
@@ -260,6 +254,16 @@ func convertJobToSecurityJob(j *config.Presubmit, dropLabels sets.String, defaul
 				},
 			},
 		)
+
+		// ensure decorated jobs upload to the private bucket
+		if j.Decorate {
+			j.DecorationConfig.GCSConfiguration.Bucket = "kubernetes-security-prow"
+		}
+
+		// de-dupe default utility images config so verify checks pass
```
Comment (BenTheElder) [inline config/jobs/kubernetes-security/genjobs.go]: this was necessary to fix `hack/verify-config.sh`, it also nets us a smaller files so :man_shrugging:

**Comment by BenTheElder** [comment]
If we start using pipelines or any other agents for kubernetes there will probably also need to be similar changes. That doesn't appear to be an issue yet.

**Pull request merged**